### PR TITLE
[Removed] [XP-47] support for e.firma for SAT institution links

### PR DESCRIPTION
**Pull request by blv-anisa**
:question: What
---

Removed support for e.firma for SAT institution links

:speech_balloon: Why
---

 
Product will drop support for e.firma on SAT links based on low usage.

### CODE REVIEW

**Review by blv-albertmaristany — COMMENTED** [review]
Same thing here @blv-anisa. Can we check if `Utils.read_file_to_b64` is being used elsewhere?

**Comment by blv-anisa** [comment]
> Same thing here @blv-anisa. Can we check if `Utils.read_file_to_b64` is being used elsewhere?

Done. Apparently, read_file_to_b64 is a method exclusive to e.firma. @AlbertMaristany, thanks for the heads up :shipit:

File: lib/belvo/utils.rb [hunk 1]
```diff
@@ -2,18 +2,6 @@
 
 require 'base64'
```
Comment (blv-albertmaristany) [inline lib/belvo/utils.rb:3]: Is this needed anymore?

**Review by blv-albertmaristany — APPROVED** [review]
Added one last comment regarding `base64` not being used, but looks good to me! 🧹

**Pull request merged**